verilog/CST/module.cc

```diff
@@ -97,18 +110,23 @@ const TokenInfo* GetModuleEndLabel(const verible::Symbol& s) {
   return &module_name.get();
 }
 
-const verible::SyntaxTreeNode& GetModuleItemList(
+const verible::SyntaxTreeNode* GetModuleItemList(
```

Comment:
(no action required)

Eh, it'd be nice to use something like absl::optional :).